config.cmake

```diff
@@ -322,7 +322,7 @@ endif()
 if(NOT DEFINED CMAKE_BUILD_WITH_INSTALL_RPATH)
   set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
 endif()
-if(NOT CMAKE_INSTALL_RPATH)
+if(NOT DEFINED CMAKE_INSTALL_RPATH)
   set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
Are we sure the install is always in **lib** and not in **lib64** in some systems ? Maybe we could use the `CMAKE_INSTALL_LIBDIR` here'

Reply:
good catch. @pierre-guillou what's your take? I remember that's been a difficulty at some point for the deb packages.

Reply:
+1 for using `CMAKE_INSTALL_XYZPATH` everywhere.
For example, the conda build requires non-standard paths such as `
    -DCMAKE_INSTALL_LIBDIR="Library/lib"
    -DCMAKE_INSTALL_BINDIR="Library/bin"
    -DCMAKE_INSTALL_INCLUDEDIR="Library/include" ` etc.

There is one more place with a hard-coded path here:
```patch
diff --git a/CMake/BaseCode.cmake b/CMake/BaseCode.cmake
index 01c94cf9..4b4dc8a5 100644
--- a/CMake/BaseCode.cmake
+++ b/CMake/BaseCode.cmake
@@ -47,7 +47,7 @@ function(ttk_add_base_library library)
     )
 
   if(ARG_HEADERS)
-    install(FILES ${ARG_HEADERS} DESTINATION include/ttk/base)
+    install(FILES ${ARG_HEADERS} DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/ttk/base")
   endif()
 endfunction()
```